slack/web/client.py

```diff
@@ -2154,3 +2154,66 @@ def views_publish(
         else:
             kwargs.update({"view": view})
         return self.api_call("views.publish", json=kwargs)
+
+    def workflows_stepCompleted(
+        self,
+        *,
+        workflow_step_execute_id: str,
+        outputs: dict = {},
```

Comment:
Let's remove the default parameter or use `None` instead. for your reference: https://stackoverflow.com/a/26320917

This is the same for other dict/list args